tests/test_webhook.py

```diff
@@ -99,16 +104,17 @@ def test_success_on_ping_event(self):
         self.assertEqual(rv.status_code, 200)
         self.assertIn('pong', rv.data)
 
-    def test_fails_on_action_not_opened(self):
-        """POST with action different of opened fails."""
+    def test_fails_on_not_known_action(self):
```

Comment:
nit: `test_fails_on_unknown_action` would read a bit more natural (and match the method docstring)